paasta_tools/cli/utils.py

```diff
@@ -352,7 +352,7 @@ def validate_service_name(service, soa_dir=DEFAULT_SOA_DIR):
     return True
 
 
-def list_services():
+def list_services(**kwargs):
```

Comment:
@solarkennedy they're required since we need the `lazy_choices_completer` to pass `kwargs` to its inner methods.